include/RemotePlugin.h

```diff
@@ -189,13 +189,11 @@ private slots:
 	void processErrored(QProcess::ProcessError err );
 } ;
 
-
-LMMS_EXPORT inline std::string QSTR_TO_STDSTR(QString const& qstr)
+inline std::string QSTR_TO_STDSTR(QString const& qstr)
```

Comment:
Why removed the `LMMS_EXPORT` ?

Reply:
This function is defined in a header file, so doesn't need to be exported in order to be called from plugins, since any code including this header will have access to the function definition. I had to remove it as MinGW emits a warning when combining `__declspec(dllimport)` and `inline`.

Reply:
Okk